spyder/utils/qthelpers.py

```diff
@@ -13,6 +13,8 @@
 import os.path as osp
 import re
 import sys
+if sys.platform == "darwin":
+    import applaunchservices as als

```

Comment:
Please move this import below the `qtpy` ones because it's a third-party library.